MC/config/QC/json/its-mc-tracks-qc.json

```diff
@@ -0,0 +1,65 @@
+{
+  "qc" : {
+    "config" : {
+      "database" : {
+        "implementation" : "CCDB",
+        "host" : "ccdb-test.cern.ch:8080",
+        "username" : "not_applicable",
+        "password" : "not_applicable",
+        "name" : "not_applicable"
+      },
+      "Activity" : {
+        "number" : "42",
+        "type" : "2",
+        "provenance": "qc_mc",
+        "passName": "passMC",
+        "periodName": "SimChallenge"
+      },
+      "monitoring" : {
+        "url" : "infologger:///debug?qc"
+      },
+      "consul" : {
+        "url" : "http://consul-test.cern.ch:8500"
+      },
+      "conditionDB" : {
+        "url" : "ccdb-test.cern.ch:8080"
+      }
+    },
+    "tasks" : {
+      "ITSTrackSimTask" : {
+        "active" : "true",
+        "className" : "o2::quality_control_modules::its::ITSTrackSimTask",
+        "moduleName" : "QcITS",
+        "detectorName" : "ITS",
+        "cycleDurationSeconds" : "30",
+        "maxNumberCycles" : "-1",
+        "dataSource_comment" : "The other type of dataSource is \"direct\", see basic-no-sampling.json.",
+        "dataSource" : {
+          "type" : "dataSamplingPolicy",
+          "name" : "tracksim"
+        },
+        "location" : "remote",
+        "taskParameters" : { }
+
+      }
+    }
+ },
+
+         "dataSamplingPolicies" : [
+           {
+             "id" : "tracksim",
+             "active" : "true",
+             "machines" : [],
+             "query" : "tracks:ITS/TRACKS/0;mstruth:ITS/TRACKSMCTR/0;compclus:ITS/COMPCLUSTERS/0;mcclustruth:ITS/CLUSTERSMCTR/0",
+             "samplingConditions" : [
+               {
+                 "condition" : "random",
+                 "fraction" : "1",
```

Comment:
If you use 100% of data, please use the direct connection. https://www.altituderando.com/Dome-de-Vaugelaz-2225m-et-Petit-Chatelet-2461m-par-les-Pars